charts/piraeus/crds/piraeus.linbit.com_linstorcontrollersets_crd.yaml

```diff
@@ -32,47 +32,33 @@ spec:
               type: string
             dbCertSecret:
-              description: DBCertSecret is the name of the kubernetes secret that holds the CA certificate used to
-                verify the datatbase connection. The secret must contain a key "ca.pem" which holds the certificate in
-                PEM format
-              type: string
+              description: DBCertSecret is the name of the kubernetes secret that
+                holds the CA certificate used to verify the datatbase connection.
+                The secret must contain a key "ca.pem" which holds the certificate
+                in PEM format
               nullable: true
-            dbUseClientCert:
-              description: Use a TLS client certificate for authentication with the database (etcd). If set to true,
-                `dbCertSecret` must be set and contain two additional entries "client.cert" (PEM encoded)
-                and "client.key" (PKCS8 encoded, without passphrase).
-              type: boolean
-            luksSecret:
-              description: Name of the secret containing the master passphrase for LUKS devices as `MASTER_PASSPHRASE`
               type: string
-              nullable: true
-            sslSecret:
-              description: Name of k8s secret that holds the SSL key for a node (called `keystore.jks`) and
-                the trusted certificates (called `certificates.jks`)
-              nullable: true
+            dbConnectionURL:
+              description: DBConnectionURL is the URL of the ETCD endpoint for LINSTOR
+                Controller
               type: string
+            dbUseClientCert:
+              description: Use a TLS client certificate for authentication with the
+                database (etcd). If set to true, `dbCertSecret` must be set and contain
+                two additional entries "client.cert" (PEM encoded) and "client.key"
+                (PKCS8 encoded, without passphrase).
+              type: boolean
             drbdRepoCred:
-              description: DrbdRepoCred is the name of the kubernetes secret that
-                holds the credential for the DRBD repositories
-              type: string
-            controllerImage:
-              description: controllerImage is the image (location + tag) for the LINSTOR
-                controller/server container
-              type: string
-            linstorHttpsControllerSecret:
-              description: Name of the secret containing the java keystore (`keystore.jks`)
-                used to enable HTTPS on the controller. The controller will create
-                a secured https endpoint on port 3371 with the key stored in `keystore.jks`.
-                The keystore must be secured using the passphrase "linstor". Also needs to
-                contain a truststore `truststore.jks`, which will be used to authenticate
-                clients.
+              description: DBCertSecret is the name of the kubernetes secret that
```

Comment:
This description doesn't belong with the key `drbdRepoCred`.

I will stop reviewing here and let you reconcile the differences between the CRDs are they were and these newly generated ones. This yaml differ https://eloquent-hodgkin-f52b42.netlify.app/ seems to work pretty well. The result doesn't have to be identical to the original, but we should make sure we're not making anything worse.

Reply:
fixed, result is identical for validation purpose. Updated the descriptions with whatever seemed more appropriate from the old CRDs or go comments